_pages/dev/seo/structured-data.md

```diff
@@ -21,50 +21,43 @@ Structured Data provides a data structure for the web that makes the web content
 
 ## Tag Managers
 
-The Structured Data let you add various schemas to the HTML, without using a tag manager solution. While tag managers have great advantages over tags implemented by a development team, there are also certain disadvantages regarding the runtime performance and the governance of tags.
+It is possible to use structured data in combination with a tag management system (TMS), but you can also use structured data to add various schemas to your HTML without using a TMS. For example, tag managers have great advantages over tags implemented by a development team, but there can also be certain disadvantages regarding the runtime performance and the governance of tags. Of course, the use of structured data is also optional.
 
-Structured Data is optional for your project. You could even consider a combination of a tag management solution and the structured data generated in Spartacus.
+For more information, see [Tag Management System]({{ site.baseurl }}{% link _pages/dev/tag-management-system.md %}).
 
-## Schema types
+## Schema Types
 
-There are various schemas for the web, and schemas can be extended as well. The schemas are maintained by the schema.org community. THe following are common schemas for commerce:
+There are various schemas for the web, and schemas can be extended as well. The schemas are maintained by the Schema.org community. The following are common schemas for commerce: WebSite, Organization, BreadcrumbList, Product, Reviews, Offer, and Rating.
```

Comment:
There are various standard schemas for the web which are maintained by the Schema.org community. Common schemas to use on an ecommerce website are:  WebSite, Organization, BreadcrumbList, Product, Reviews, Offer, and Rating. Other, custom schemas might be proposed by certain crawlers.

Reply:
👍